app/components/primary_dashboard_programme_community_activity_group_component.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class PrimaryDashboardProgrammeCommunityActivityGroupComponent < CmsWithAsidesComponent
+  delegate :current_user, to: :helpers
+
+  def initialize(title:, programme_activity_group:, current_user:, aside_slug: nil)
```

Comment:
When we call this, we pass in the aside slug and title, which are just coming from the programme_acitivty_group. Could we default these to nil and if they are nil get the value from the model, rather than having to pass them in if we are using the values on the model.

Do we need the option to override them, if not we can just not bother with the nil parameters?